module/admin/helper.js

```
@@ -241,50 +247,66 @@ module.exports = class AdminHelper {
-						// Extract project template IDs from solutions
-						projectTemplateIds = solutionDetails.map((solution) => {
-							return solution.projectTemplateId
-						})
+						// Delete solutions from project service
+						await solutionsQueries.delete(solutionFilter)
+						deletedSolutions.deletedSolutionsIds.push(...solutionIds.map((id) => id.toString()))
+						deletedSolutions.deletedSolutionsCount += solutionIds.length
 
-						// Add main program ID to deletion list
+						// Audit log entries for solutions
+						solutionIds.forEach((id) => resourceIdsWithType.push({ id, type: CONSTANTS.common.SOLUTION }))
```

ESLint

[error] 'CONSTANTS' is not defined.

(no-undef)